File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_26.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// модель справочника принтеров
public class MB_S_26 {

    /// Номер ТК
    @JsonProperty("WERKS")
    @Getter private String tkNumber;

    /// Наименование принтера
    @JsonProperty("PRINTERNAME")
    @Getter private String printerName;

    /// Информация о принтере
    @JsonProperty("PRINTERINFO")
    @Getter private String printerInfo;

    public MB_S_26(String tkNumber, String printerName, String printerInfo) {
        this.tkNumber = tkNumber;
        this.printerName = printerName;
        this.printerInfo = printerInfo;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_26.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// модель справочника принтеров
class MB_S_26(
        /// Номер ТК
        @SerializedName("WERKS")
        val tkNumber: String,

        /// Наименование принтера
        @SerializedName("PRINTERNAME")
        val printerName: String,

        /// Информация о принтере
        @SerializedName("PRINTERINFO")
        val printerInfo: String
)